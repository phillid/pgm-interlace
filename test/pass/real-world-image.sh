#!/bin/sh

${EXECUTABLE} ../input/real-world-image.{0..4}.pgm > real-world-image.pgm.out
