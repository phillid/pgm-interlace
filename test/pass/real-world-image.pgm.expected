P5
100
100
255
##((((((((((((#####################(((((((((((((((#####################----(((((((((((((###################(“3---((((((((((((((###################-3LL----((((((((((((((##################ôp8ïQ833-((((((((((((((##################(-=ÖuQpB33-(((((((((((((((#################(--=LÛuÛ[333(((((((((((((((#################(----™‰fBÌzG88-(((((((((((((((################(=3----8“`BÛkf`[-(((((((((((((((################(Ñ3pB---38[[uÿÑ¼„Bk---(((((((((((((###############-3Ž8`p8--33`uÖ™£G[kB83------((((((((###############((-8ÿLê`83333„VLÁGÿÿk8833-------(((((((###############((--BLfÿž3338¼BGÌŽuÿz=8333--------(((((################(((--8VÿB£ô=338·G‰ukzêÁ=8333---------((((##################((((--8ÿp[[ô[88[z“ÿÿ²G`G3333----------((((######################(((((-33`Vzÿ„[£ÿLGÿ[G„²z­ÿ8333-----------(((################(((#######((((((-33¼žkÿuu[BÖVï[L`ÿž[fB8333----------(((#####################(83`--(((((((##((((((-38ÿÖÿŽ­ÁVÑBÿfGÆ²zz“f=8833-----------(((##################((-8-=3-(-3k8f8--((((((((((((((-38‰GGVÿ[ÿÿ8B­‰pVÆzzkG8883-----------(((####################(((--·8p8---8=VG---((((((((((((((8GfÌŽQ[G¨Q8=ÿ[ÿ£ÿz¼ÿf8883------------(((#####################((((---²„Ñ™---=QG“8--((((((((((((((8[`“ÿL­uG==·„pÿužêVG==3------------(((#######################(((((--3²Lu`8--8pÛzz83-((((((((((((((3Lzÿàïåÿ“å=3LV[[Žÿ[¼B==-------------(((#####################((((((--3BkB=338ÿÿ¨àG33-(((((((((((((=ï„u`QÆÖÌÿž3B­²„£VÖ™Ö²‰B3333333333---(((####################(((((((-33GÿBVG333„™u­ÿ83-((((((((((((-88L‰“[Ñz“ÿQ8=`™kVÿÿÿkkàLÿ=83333333333--(((###################((((((((33GžÑVô838ÿ¼u[Æ=8-((((((((((((-88B™ÿ£VïuuG==Qÿ™Æ‰·¨£V[“L=888333333333-((((#################(((((((((38GÌôLk=88êÿ™G£`B-((((((((((((338=ÿ£kffÿÿz==L`k“£™Q·zÿG=8888333333333-((((((##############(((((((((-8VG„LLÛ=Gàk­™¨pL8(((((((((((-333BÆ`ÖVÿz‰VB8V­pp­™ÿÿÖfpGB88883333338883-((((((((##########((((((((((-`ÿQLQLkz²fuÿŽ™pà`-(((((((((--3338²Lfÿ`ôêžL8LÿV£ÿkÿkuQÿGB888883333888u=--(((((((##########((((((((((-[Lzpf™`êpkLLV“„Á88-((((((((--3338GÿÖåÿÿpàf=L¼`ÆÖê¼ÿV`²ŽQ888883338==pL‰3--((((((##########((((((((((3=åÿzÿf`BôfQ[uê883--(((((---333==Žpu£ÆÿufBÿÿ`ÛÿQÆVzÿ£ÿB====88===Lÿf=8=3----((#########((((((((((-38GÿÖ‰ffp8Qž¼¼[ÿÑ=833--((((---3338BÿÆÿÿ[¼ÿôQ`ÿÑÿÛ„LÿÿÁ­ùŽBB===BVBG„ÿ²k88kz83----#########(((((((((--38=£ÿ„[fG=B“`kÿžp=333----(----3388Bžkÿ‰·fÑ­£êŽŽ¼£åVåù²Ž„fLG=BBŽÿVpÿG™B8ŽBB833---#########((((((((--333=ÿùï¨ÿ==ÌàÁ‰kG3333--------3388=ï£ÿàpkQ[fzu„ÿ²ÿÿôÿÿÿuÿùVLG[„pÿkŽG=B[ô88333--#########(((((((---3338£žuà[·BBÿ­pkÿ[p3333--------3388B¼²‰ÆÁÌÿ`ÑùufÿÿpÛÿÿÿ­Û¼²·z[ÿÿÿz==ÿL88333--#########(((((((---3338Ñ``[`fL=¨u·„Q‰G=333---------388BÁô“Qfuùÿpf“ùÿàÖkôÿÿ·ÿô`ÿÿu[ÿ[pÌ·G=GÆB883333-#########((((((----333=GÌ“™„ÌB=G·ê²ÿŽÑ=833---------38=Gk²ÿfÿŽ“VÑ‰ÿÛ‰­uQ·ÿ¼fÿÿŽufÖžkåV==„`p=833333-#########(((((-----3388GŽï™Ñ==Gÿÿ¨zfQ=883---------38=p`Æÿÿ™ê¨VQåŽÿÿfÁùpQQ“„£pu[[ÿz[ÿzG=²ÿzÿ=833333-#########(((((------388Bÿ¨Ž·ÆG=Bÿ`kz²ŽB883----------8G`žzÿQ“Q¼ŽÑïÿ“·¨QGÛkf™„ž‰£`·LBGŽ`pL=8333333#########((((-------388Lž[¨ÿÿ­8GÖu`¨ù`883----------8LfÁÿÿÿV‰uf·ÿÌŽåV„pQG‰Ì`¨V`VÿÁÿkLG¨ž£¼G=3333333#########((((-------388GÑÿ¨ÿkÑBGÑpfÆÿ‰L=88----------8·kÑVQ²Qÿff[¼`‰êQp™`G`u¼ÁŽ“Á™f“GùŽ¨žÆB=3333333#########(((---------8=BpzôkkpBBfkÿ²ÿf™B=8----------=GÿÿQQÿ[²ï[£„ÆkÿåGzÿ‰QQž¨[`V`Á·²­ÿÿzÑVVB83333333#########(((---------8=Lz[“£ÿQQG¼QQQ£ÿŽQ==---------3==GkQVÿÿfù‰ÿÿzÿïuG[ÿ“LLÖÿÿ‰žzz‰­k[`[ÿÿG33333333#########(((---------3QÿåQQpÿfÿ™„Q[Q¼[“u„ÿ3--------8==GÆžÿÿf„ÿïfÿÆïå·G[¼zLGQÛÿ²åÿ­­Æÿ“p™fk“Q33333333#########(((---------3GQÿz[V[fppk­“p`VVzGu8-------388=B™z²ÁåàQÿÿ·ÿLfÿÿGGQž·ÿfÌ·Qzf`f`‰LL33333333#########((----------=GŽÆ²ÿÿÿ„Lzÿ“ÿïêf²ÿG=3------388=Bôåÿ·ÿÿÿL²“ÿÿ‰fÿLù‰LGLùkpÿÿÿàÿÿÛù£¨ÑB83333333#########((---------3==„‰uf`[V[¼f[·f™ÿŽÿu==83----38888Gåf“ÖêôpQpžÿ™ÿôLGkÿQLVÿk¨ÿÿzÿ„ÑÿÿÛzpB=83333333#########((--------388=Gf¼ÖÿffVf`ÿ`[VÿÌ“Q==8833-338888¼ïu`“V¨ÌLQ£f‰pfQGQžQL[ÿpÿ“êpŽzÿuf[u£B=88333333#########((------33388=BÑÿ‰kÿ¨pÿÆ¨åQžu[²QB=88833338888Guÿ²QzŽGQ‰¨ÑžÿLGQÿ[QïÿÿïÿÿÿŽ¼‰ÿ[­¨BB=88333333#########((-----333888=BåêÖzVÿ‰·‰ÿÿïÛåVkÿB=8883333888=BåŽÿLŽzQLÆ“ÿpuQLL£žÛÿžÿÿÿ²kÿ[[Æ²žLB==88833333#########((----33338888GÆÆ™ÿk™å²‰“p¨ùÿfÿŽG=8888338888BBÿ¼­êGÿÿÿÿÿ££ÌÿVLVŽÿzÿÿÿÿïuf²```kåLB==88883333#########(----333338888ùÖfï`Û`Ñfÿ¼Ž‰Ö[ÿ¨`V=888833888=BGÿ¨¨ÿGùÑÿLQ£ÿ[„kQ[êk‰ÿÿÿÿÑkk`„ÿ²``GB==88888333########((---333333888=GfÆÿ¨„ÖÿÆÿzÿ„ÆpåuG888883888=BGÆÿkQ£f[GpÖ“k­kVÑkÿÿÿÿÿÛpÿ£V`ÑŽŽQG===88888833########((--333333388=BB­ÁÿzÁ·ÿ„ÿÿ£zzk­¨G=8888388===LfàÖÿŽ[ÿ[GQŽÿpÿÑêzÿÛÿÿÿžÑ™p‰fŽ‰‰ÿLG===88888833########((--33333338==BGfÿk·u[£`“ÿÁQÿ²V¨™ÿBB=888888===„™£åfLQ“àLQÿ‰““pÿ²Ñÿÿÿ£pžpÆÿÿkÿÿÿLB===88888883########((-333333338===GÿÑ“V“fu„up­Lÿ­Q“z¼GB==8888===B„kpåzLGužLLuŽkÿÿÿÿÿÿÿÌàuÁkàÖ“ÿ`­“L====88888883#######((--333333338===ÆåÁÆL­[[ÿÿ­L™fLÿŽuLG===888===LÌÿÿ‰·LLfêLQÆ²ù·zÿÿÿÿÿÿÿÑk‰[žÿÌp[L====88888888#######((-333333333==BGÆ·ÿÿLÛÿQp£ÿÌLž£LÁ¼·ÿG====88==BGuuÿÿïQL`žVVÆŽzÿêÿÿÿÿÿ„ÛŽÿ„ÿ[™pÿÿ`¨====88888888######(((-388833338BBBG“fÿÿQÿÖQÿ„à·GukGÿ¨z·L=======BGV„`™kÿQLÿÿ`ffïÿÿÿÿÿÿÿÿ­kÿŽ·pf²ž·QB===88888888######((-888888888=GBGQÿkÿ‰QÿàÆ¨Ûÿ·Là“Qfuÿÿ£LB====BBLk`ÿ[[Û`Lÿ“™“Û“ÿÿÿÿÿÿÿÿÌ[[ÿz™“¨ô‰k„LBB==88888888#####(((-“B88====BGVfÿ„VêùÿLÿÿL­ÿ·Æ`¨ŽkkÿÿzÿLGBBBBBGLàÿÌp‰Ì[VÿÛpÿÿÌÿÿÿÿÿÿzÿ¨[VÿÿÿàŽ™[GBB==88888888###((((-8ÁkVB===BBL™ùåkÿuÆ¼G²ïLÿÿpŽV[ÿ[[£ÿŽÿVGGBBBBLk™ÿÿ™ÿ„Ž`ŽÿÿÿÿÿÿÿÿÿÆÿÑ£¼VV`ÿ£™ÿÿfôQGBB==88888888##((((-8=BpàÖGBBGBfk`ÿÑkŽ“žL`„GŽ“ÿQQ‰QQÿÿÿÿÿQLGBBQ[êV`ÿÿ¼žÌ™Æ“ÿÿÿÿÿÿÿÿ£uŽ™kVQVïÿzù“ÿ¼QGBBB=88888888(((((-3Ûå=Q£VÿÑLGL[ÿÿÑzåÿ`àL[ÿLkz[ÿ[L²zGÿÿÿùuLQ™Qp‰£ÿÿ[`ÿÿùÿÿÿÿÿÿÿÿÿÿ‰ùuÖÿfQQVzÿ¨kÑz™GGBBB=88888888(((--38BêkBB`‰£ÿåÿžô™ÿÿ“V`åL`ÿfÿQQ„VL²`kÿÛfÿzÛÿ™kÿufÿuÿÿÿÿÿÿÿÿÿÿÿÿz²ÿŽfLQV„ÿ™p¨ÑkGGBBB=888888==(---3388[VVBG‰£ôï“ÿÿÿÿ“‰ÿzÿ™ÆV`Ûÿ²[LåVLÿk`ÿ‰fkkÿÿÿÿÿÿÿ¨ž“ÿÿÿÿÿÿÿÿÿÿÖ¨upÿ­ÿÿQQVz¼Ž²pÿLGBBB=88888===---33388BÌžLBLÿkÿÿÿfÿ­žÿÛ­åÿÿ·uÑÿ„Ì`Lÿ­Lÿÿuïkzï‰ÿÿÿÿÿÿÿÿ‰ÿÿÿÿÿÿÿÿÿÿÿ‰êp£kÿÿfQQpÿÿzÛÛÿkLGBBB=888=====---33388=BÿÿL=LÿÑ`ÿ£­²zŽÿ“ÿÿÿÿÿžÌfpÿVÿÿVÿÿk`ÿf“ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ£‰ÿ¨ÿ``²`QQfÆ‰²upÿ[LBBBB=88======--333388=BLôŽŽGLÿf“fÿ™¨Öàÿÿÿÿÿÿÿÿÿÿ„ÿÿ[ÿÿ[pÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ­ÿuÿŽÿV`„`QLVÿÿÑu£[LBBGG=========-3333338==GzÿfLGLfÿVkÿzzêùÿÿÿÌ“ÿÿÿÿÿÿÿÿÿuu„™ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÖÿ·uÿŽpV™ÛfVLQ·ÿ‰kÁÿ[QGGGLB========33333338===[ïÿÿGGLzå¨åÿÿ­Ìÿp™ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÑÿ²­ÁV[uÿVQVÿÿÖ·ÿÿ[[LGGVGBBB=====33333333===Gÿ·ÿzLLLÿ·ÿÿÿÿôÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÛ“¼ÁÛ¼ÿ`VV¨žpQVÿàù­Ì„VuLGGVGBBB====B333333338==GGÿ™p[LLV„ÿÿÿÿ²ÿÿÆÛÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÌÿÿÿ¼àåÿfVVÿ£fQV²¨“™ÿù`QLGQ[GBBBB===G333333338==BGÿ[fÿkQQ`ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ™“uzÿÁ¨¼VVf“[QfÿÿïŽÑÿQLL[QGBBBBB==L333333338=BBGQ`ÿÌ¨[VVÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÑÿÆu‰ÿ¼êÿVQfù`VŽÿ­êÌŽŽÿQLQùGGBBBBBBGL333333333=BBBLz`ï„„``ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ­™ÿÿÿ·¨ÿÿ`VQ²“`Vÿÿÿù“ïzÁQQ`êGGBBBBBBQ[333333333=BBBQ`fÿÿÁ‰ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ²ÿužùÿô`QVÿåk`ÿÿ“ÿŽpfVQ£ÑGGBBBBGGVÌ333333333=BBBV`ufÿfkÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ¼Ö„ÿuÿÿÑ¨„`QVŽïÿÿÿ­¨ÿÑ£ŽfVVÿ£GGBBBGGL[™333333333=GGL`ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ·™„zÿ‰ùïÿ`Q[zÖÿÿÿŽ„ÿŽÿ`V[åfGGBBBGGLžÿ333333338[VLQQ“ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÖÿÿåuÿÿ­­ÿÁ[V„fÿÿÿfu‰Æ„u[`·VLGBBGGGLkê33333338=Gÿ·V[ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿž™·‰Áÿ·™­Áf[Ñkÿž“`£ÿÿà„Ñ`p‰VLLBGGGGLfž8888888==BLÖôkÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿå¨Öÿÿ„êzÿÆ¨¨[fuÿ²Vk„­ÿ„‰ppÿzQLLBGGGLQfz888888==BBL[fÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿôÿÿ£Áÿù¨pÿ­­ÿkf`“ÿùô[kÑ“ÿôêzf­ÿQLLGLGGLQpu8888====BBGQÿfzÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿåž¨ÌÿÿêÛïàÿkÿ[pŽÿÿ[kÿÿ‰ž²žkkQLLLLGGQVuÿ888====BBBBQ[ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿž²ÿÿ™zÿžÿÿåkê[ÿŽÿu[­·ô­à­ÿÿ`QLQQLLGQV£ï88=====BBBBQV™kÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ²ÿzÿÿzpÿïÿ™pà[uÿp`Æ“žà„¼êÿÿ`QLQQLLLQVÿÆ8======BBBBQQkÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ“à¼ukÑŽÿ·uu`uÿpkÿ™ÿ‰Áå¨ê`VLQQLQVQ[êÿ======BBBBG``p²ŽÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÆÿÿÿzêŽup™ÑêÌk`ŽÿzfŽÌÿ™Æàÿÿ­`VVfQLQVQ[¼ïBB====GGGLQ[`Áÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿï¼·ÿ·‰ÿ‰zzÿÿÿÿ„kÿ£ÌžfžÿÛ™ÿŽÿ„ÿfVVuVQQVV`à™BBBGBGGGLLQ[uÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ‰ÿô“uêÿåÛžpÑ“žÖzf“·ÿ™‰“ôïàk[[pVV[fkfÿ¨BGGLQLLGQLVk·™ÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÑÿŽÿÿŽŽuÿÛåÿÿ‰²¨ÿzk‰ÿÿÿ‰²£Ñ“p``u`[[f¨kÿÿf[[`kuu[zk¨ÿÛÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿ£ùÿ­¨‰ÿÿïÿÿž™ÿàÿŽŽ²ÿÿÿ£ùàÿ£„zz“uuuÆ²ÿÿ========BBBBGLV`[QQQ[`fzkpp‰Ž­¼êàÿêÛÿÛêùÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿÿu`[VVQLLLLGGGGGLLGGGBBBBBGGQGBBBBBGGGL[f888888888888=LGGGBBBBBGLQQQQ`kk``kpuppz„‰„žž·Öÿÿÿÿÿÿÿÿÿÿÿÿÿÿpf[QQLLGGBBBBB==========8888888=======G[3333333333333338==BBBBBBGLLLQQQQQQV[[kf````kuzŽ­àÿÿÿÿÿÿÿÿÿÿzf`[QBBBBBBB========88888888888888888==G-33333333333333338====BBGGGLLLGGGBBBBGLVVVV[[``pp„Ž£¼ÿÿÿÿÿžk`QQLGBBBB=====8888888888888888833333--------33333333333388======GLGGGBBBBBBBBBBBBGQQQQVfkppp„zffVQVVVLGBB=88833333333333333333333------------------3333333888==BGG==========BBBBBBBBGLLQQQ[V[``k```[VLGGGGGB8333333333333333333333----------((((((((((------3338888GB================BBBBBGLLQQQQQQVVQQQGBBBBB=83333333333333333333-------------(########((((((((((--8888888888888833338888==BBGLLLLLGBBBBBGGBBBB==83333333333333333----------------